### PR TITLE
Adds jax-ws compatibility to docs

### PR DESCRIPTION
**Pull request by andrewsouthard1**
Adds jax-ws integration from 0.76.0 to docs: https://github.com/DataDog/dd-trace-java/pull/2473

https://docs-staging.datadoghq.com/andrewsouthard1-patch-1/tracing/setup_overview/compatibility_requirements/java/

### CODE REVIEW

**Review by kayayarai — APPROVED** [review]
Ping me when you are ready for a merge (assume you're waiting on the other approval)

**Review by mcculls — APPROVED** [review]

**Pull request merged**
